text/0002-data-aggregator.md

```diff
@@ -0,0 +1,72 @@
+- Feature Name: data_collector
+- Start Date: 2021-07-05
+- RFC PR: (leave this empty)
+- Hathor Issue: (leave this empty)
+- Author: Giovane Costa <gigio.coder@gmail.com>
+
+# Summary
+[summary]: #summary
+
+Data aggregator will aggregate all network data from Redis, check if something changed and update if necessary.
+
+# Motivation
+[motivation]: #motivation
+
+We want to have an overview of all the hathor network. Today we have only a single node point of view. 
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+Data aggregator will get data collected by `data-collector`, aggregate them into a Network dataclass, compare with saved data and save it if different.
+
+# Reference-level explanation
+[reference-level-explanation]: #reference-level-explanation
+
+Three new dataclasses will be created: `AggregatedNode`, `AggregatedPeer` and `Network`.
+
+- `AggregatedNode` and `AggregatedPeer` will be a bit smaller version of `Node` and `Peer` respectively, having `up_since` as the main change.
```

Comment:
Nice. Add a description of each field and how they are aggregated, please.

Reply:
Added